src/api/aux.cpp

```diff
@@ -22,6 +22,9 @@ void parse_json_file(std::vector<Token> &tokens, std::vector<Order> &orders, std
     
     for (auto &i: json_file["tokens"].items()) {
         boost::multiprecision::mpf_float t;
```

Comment:
Sugestion: rename "t" as external_price, or "ep", or something that is easy to remember.

Reply:
Done